lib/phoenix/live_dashboard/system_info.ex

```diff
@@ -63,6 +63,10 @@ defmodule Phoenix.LiveDashboard.SystemInfo do
     :rpc.call(node, __MODULE__, :applications_info_callback, [search, sort_by, sort_dir, limit])
   end
 
+  def fetch_applications_2(node) do
+    :rpc.call(node, __MODULE__, :applications_info_callback_2, [])
+  end
```

Comment:
I believe this and below were added by accident? :)

Reply:
Sure xD